src/lib/containers/LoadingScreen.tsx

```diff
@@ -38,11 +38,11 @@ type BlockingLoaderProps = {
 }
 export const BlockingLoader: React.FC<BlockingLoaderProps> = ({ show }) => {
   useEffect(() => {
-    document.body.style.cursor = 'wait'
+    document.body.style.cursor = show ? 'wait' : 'default'
     return () => {
       document.body.style.cursor = 'default'
     }
-  }, [])
+  }, [show])
```

Comment:
We were changing the spinner anytime this component was rendered, even if the loader isn't actively being shown. Very confusing, since it looks like Synapse is loading when nothing is actually happening.